docs/proposals/20230407-managed-k8s-capi-contract-changes.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Contract Changes to Support Managed Kubernetes
+authors:
+  - "@jackfrancis"
+reviewers:
+  - "@richardcase"
+  - "@pydctw"
+  - "@mtougeron"
+  - "@CecileRobertMichon"
+  - "@fabriziopandini"
+  - "@sbueringer"
+  - "@killianmuldoon"
+  - "@mboersma"
+  - "@nojnhuh"
+creation-date: 2023-04-07
+last-updated: 2023-04-07
+status: provisional
+see-also:
+  - "/docs/proposals/20220725-managed-kubernetes.md"
+---
+
+# Make Cluster Infra Resource Optional
```

Comment:
nit: Let's also update this title and the one in l.30

Reply:
thx!